index.html

```diff
@@ -1,37 +1,495 @@
+                    <li class="footer__topics__item"><a href="/category/inspiration">Inspiration</a></li>
+                    <li class="footer__topics__item"><a href="/category/ios">iOS</a></li>
+                    <li class="footer__topics__item"><a href="/category/javascript">JavaScript</a></li>
+                    <li class="footer__topics__item"><a href="/category/mobile">Mobile</a></li>
+                    <li class="footer__topics__item"><a href="/category/pattern-libraries">Pattern Libraries</a></li>
+                    <li class="footer__topics__item"><a href="/category/performance">Performance</a></li>
+                    <li class="footer__topics__item"><a href="/category/photoshop">Photoshop</a></li>
+                    <li class="footer__topics__item"><a href="/category/plugins">Plugins</a></li>
+                    <li class="footer__topics__item"><a href="/category/react">React</a></li>
+                    <li class="footer__topics__item"><a href="/category/responsive-web-design">Responsive Web Design</a></li>
+                </ul>
+                <ul class="footer__topics__list1 flex-container flex-row  align-items">
+                    <li class="footer__topics__item"><a href="/category/service-workers">Service Workers</a></li>
+                    <li class="footer__topics__item"><a href="/category/sketch">Sketch</a></li>
+                    <li class="footer__topics__item"><a href="/category/typography">Typography</a></li>
+                    <li class="footer__topics__item"><a href="/category/ui">UI</a></li>
+                    <li class="footer__topics__item"><a href="/category/usability">Usability</a></li>
+                    <li class="footer__topics__item"><a href="/category/user-experience">User Experience</a></li>
+                    <li class="footer__topics__item"><a href="/category/wallpapers">Wallpapers</a></li>
+                    <li class="footer__topics__item"><a href="/category/web-design">Web Design</a></li>
+                    <li class="footer__topics__item"><a href="/category/wordpress">WordPress</a></li>
+                    <li class="footer__topics__item"><a href="/category/workflow">Workflow</a></li>
+                </ul>
+            </div>
+        </div>
+
+
+
+        <div class="footer__notes bg-color">
+            <p>With a commitment to quality content for the design community.<br>
+
+
+                Founded by Vitaly Friedman and Sven Lennartz. <span class="lining-figures">2006–2020</span>.<br>
+
+
+                Smashing is proudly running on <a href="https://www.netlify.com/?utm_source=link&amp;utm_medium=parter&amp;utm_campaign=sm-footer">Netlify</a>.<br>
+
+
+                Fonts by <a href="http://latinotype.com/">Latinotype</a>.</p>
+        </div>
+        <div class="inline-flex bg-color">
+            <!--            <img class="laptop" src=" /smashing_magazine_clone/images/svg/laptop__left.svg" alt="">-->
```

Comment:
It’s better to delete commented code. If you need to review a previous version of your code, use git history.

![comments](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png)